services/rds/src/test/java/software/amazon/awssdk/services/rds/PresignRequestHandlerTest.java

```diff
@@ -55,7 +55,8 @@
     private static final Region DESTINATION_REGION = Region.of("us-west-2");
 
     private static RdsPresignInterceptor<CopyDbSnapshotRequest> presignInterceptor = new CopyDbSnapshotPresignInterceptor();
-    private final CopyDbSnapshotRequestMarshaller marshaller = new CopyDbSnapshotRequestMarshaller(new AwsQueryProtocolFactory());
+    private final CopyDbSnapshotRequestMarshaller marshaller =
+        new CopyDbSnapshotRequestMarshaller(AwsQueryProtocolFactory.builder().build());
```

Comment:
Should we just have a create method?

Reply:
We need to add overload `create` methods if we add state to this class, so i avoided it. We don't use it in many places, so it should be fine. We can always add it later.

Reply:
Sounds good.